### PR TITLE
Smaller graphics fixes

### PR DESCRIPTION
**Pull request by heikoklein**
This PM allows SnapPy to plot with a newer bdiana version with de jure rather than de facto country-borders. In addition, country-codes are plotted again, and a test-dataset is removed.

PS: Sorry, the branch-name is completely misleading, this branch has nothing to do with meps.

### CODE REVIEW

File: utils/SnapPy/snapEnsPlot.py [hunk 1]
```diff
@@ -23,9 +23,29 @@
...
 def plotMap(data, x, y, ax, title="", title_loc="center", clevs=[10,100,300,1000,3000,10000,30000,100000, 300000, 10000000], colors=None, extend='max'):
-    ax.add_feature(cartopy.feature.GSHHSFeature(scale='low', facecolor='none', edgecolor='whitesmoke', linewidth=.2), zorder=100)
-    ax.add_feature(cartopy.feature.BORDERS, edgecolor="lightgray", linewidth=.5, zorder=100) 
-    #ax.gridlines(draw_labels=True)
+    ax.set_facecolor("#f2efe9") # background, osm-land
+    ax.add_feature(cartopy.feature.NaturalEarthFeature(
+            category='physical',
+            name='ocean',
+            scale='50m',
+            facecolor="#aecfe0"),edgecolor="none", zorder=10) # #aecfe0 = osm-ocean
```
Comment (magnusuMET) [inline utils/SnapPy/snapEnsPlot.py:31]: weird formatting

File: utils/SnapPy/snapEnsPlot.py [hunk 2]
```diff
@@ -23,9 +23,29 @@
...
+    # ax.add_feature(cartopy.feature.NaturalEarthFeature(
```
Comment (magnusuMET) [inline utils/SnapPy/snapEnsPlot.py:32]: Remove commented code

**Review by magnusuMET — APPROVED** [review]
Nothing seems obviously off. Might be an idea to make a new `DEFAULT_BASEENV` to make sure the newly installed `bdiana` takes presedence over the old one (although should be ok)

File: utils/SnapPy/snapEnsPlot.py (hunk 1 above)
Comment (heikoklein) [reply utils/SnapPy/snapEnsPlot.py]: won't fix

**Comment by heikoklein** [comment]
Adding some smaller snap-fimex interoperability stuff, requiring fimex-1.6 (current default is 1.9)

**Pull request merged**